source/whats-new.txt

```diff
@@ -20,9 +20,26 @@ What's New
 Learn about new features, improvements, and fixes introduced in the
 following versions of the {+driver-long+}:
 
+* :ref:`Version 5.1.2 <javars-version-5.1.2>`
 * :ref:`Version 5.1.1 <javars-version-5.1.1>`
 * :ref:`Version 5.1 <javars-version-5.1>`
 
+.. _javars-version-5.1.2:
+
+What's New in 5.1.2
+-------------------
+
+The 5.1.2 driver patch release includes the following changes:
+
+- Fixes an issue that prevents the driver from encoding and
+  decoding concrete classes that extend generic base classes, in cases
+  that you specify the base class as the generic type of the ``MongoCollection`` instance.
+
+- You can validate signatures of the driver artifacts published on
```

Comment:
S: so the bullet point starts with a verb:
```suggestion
- Allows you to validate signatures of the driver artifacts published on
```

Reply:
removed this item per Slack discussion